docs/source/conditions.rst

```diff
@@ -252,6 +252,15 @@ Here's the list of pull request attribute that can be used in conditions:
    * - ``title``
      - string
      - The title of the pull request.
+   * - ``time``
+     - string
+     - A time in format ``HH:MM``.
+
+
+.. important::
+
+   ``time`` does not support ``~=``, ``=`` and ``!=`` operators.
```

Comment:
It makes a bit sad that we don't support `=` TBH. :)

Reply:
Yep, which circle back to the issue I was mentioning here https://github.com/Mergifyio/mergify-engine/pull/2695#pullrequestreview-677099915